### PR TITLE
ORCA_Blog_Week7

### PR DESCRIPTION
**Pull request by YiqinAliciaShen**
New Blog Post

### CODE REVIEW

**Comment by YiqinAliciaShen** [comment]
@arokem Ready for merge.

File: _posts/2016-07-29-orca-week-7.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<iframe width="650" height="600" src="{{site.url}}/assets/images/zonal_stats_alicia.html" ></iframe>
+
+While this is not a direct comparison, since ORCA data include the transit data of all individuals (not only workers) commuting for all purposes. But as we can see from this map, there are discrepancies for some tracts. A more direct comparison between ORCA and APC would be available after our merge attempt.
+
+### Bias #2 APC vs. ridership
+
+Since for King County Metro  only a proportion of the buses have APC counts, in order to be able to use APC as our ground truth, we need to understand whether APC is representatively sampled across space and time. 
+
+From talking to project lead and transit agencies, we learned that APC sensors, which are attached to buses themselves, are never moved. Instead, buses with sensors are dispatched on different routes in an effort to get ridership estimates for all routes for the transit agencies (KCM). We also learned that this strategic dispatch happens mostly during weekdays.
```
Comment (arokem) [inline _posts/2016-07-29-orca-week-7.md]: "...talking to project lead..." => "...talking to the project lead..."

**Comment by arokem** [comment]
Looks good. One small comment

**Comment by YiqinAliciaShen** [comment]
Thanks for the catch!

**Comment by arokem** [comment]
Why did you make that recent change? If you make the change I suggested, I can merge this.

**Comment by YiqinAliciaShen** [comment]
@arokem ready for merge again!

**Pull request merged**